routes/api/v1/playlists.js

```diff
@@ -84,10 +84,43 @@ router.delete('/:id', (request, response) => {
   });
 })
 
+router.get('/:playlistId/favorites', (req, res) => {
+  var playlistId = req.params.playlistId;
+  return database('playlists')
```

Comment:
Would like to refactor during a later sprint.

Reply:
Definitely could refactor all three database calls into helper methods at a later time